ci/ReleaseTool/Common.cs

```diff
@@ -10,6 +11,70 @@ internal static class Common
     {
         public const string RepoUrlTemplate = "git@github.com:{0}/{1}.git";
 
+        // Names of the version files that live in the UnrealEngine repository.
+        public const string UnrealGDKVersionFile = "UnrealGDKVersion.txt";
+        public const string UnrealGDKExampleProjectVersionFile = "UnrealGDKExampleProjectVersion.txt";
+
+        // Plugin file configuration.
+        public const string PluginFileName = "SpatialGDK.uplugin";
+        public const string VersionKey = "Version";
+        public const string VersionNameKey = "VersionName";
+
+        // Changelog file configuration
+        public const string ChangeLogFilename = "CHANGELOG.md";
+        public const string ChangeLogReleaseHeadingTemplate = "## [`{0}`] - {1:yyyy-MM-dd}";
+
+        // Unreal version configuration
+        public const string UnrealEngineVersionFile = "ci/unreal-engine.version";
+
+        public static bool UpdateVersionFilesWithEngine(GitClient gitClient, string gitRepoName, string versionRaw, string versionSuffix, string engineVersions, NLog.Logger logger)
+        {
+            return UpdateVersionFiles_Internal(gitClient, gitRepoName, versionRaw, logger, versionSuffix, engineVersions);
+        }
+
+        public static bool UpdateVersionFilesButNotEngine(GitClient gitClient, string gitRepoName, string versionRaw, NLog.Logger logger)
+        {
+            return UpdateVersionFiles_Internal(gitClient, gitRepoName, versionRaw, logger);
+        }
+
+        private static bool UpdateVersionFiles_Internal(GitClient gitClient, string gitRepoName,  string versionRaw, NLog.Logger logger, string versionSuffix = "", string engineVersions = "")
```

Comment:
This is the main point of the refactor, to share the update version code between prep and prepfull commands.

Reply:
This _Internal business is not the most elegant but the best I could come up with while still sharing code. Hopefully the explicit naming of the UpdateVersionFilesWithEngine and UpdateVersionFilesButNotEngine make this API explicit at least (instead of relying on filling optional arguments).